src/database/append_smiley.rs

```diff
@@ -0,0 +1,61 @@
+use crate::database::models::{Restaurant, SmileyReport};
+use serde::Serialize;
+
+#[derive(Debug, Clone, PartialEq, Serialize)]
+pub struct RestaurantWithSmileyReport {
```

Comment:
Kunne du ikke lave en "øvre" struct på Restaurant som ikke bare er en ren kopi af "Restaurant"? Så fx.
```rust
struct RestaurantWithVec {
    pub restaurant: Restaurant,
    pub smileyreports: Vec<SmileyReport>,
}
```

Reply:
man kunne nok gøre dette, men så ændres hele måden at du tilgår restaurant data på. så i steden for at gør restaurantwithsmileyreport.field, skal man gør restaurantwithvec.restaurant.field

græver en omskrivning af kode rigtig mange steder

Reply:
det vil også gør at det json som sendes til frontend også vil have en anden struktur end hvad vi snakkede om i starten